packit/utils/lookaside.py

```diff
@@ -0,0 +1,51 @@
+import configparser
+
+from pathlib import Path
+from typing import Dict, List, Union
+
+import pyrpkg
+
+from packit.exceptions import PackitLookasideCacheException
+
+
+def get_lookaside_sources(
```

Comment:
I wonder if and how we want to test this function.

Reply:
Well, I suppose I could flexmock `pyrpkg.lookaside.CGILookasideCache` and `pyrpkg.sources.SourcesFile`, but then there is not much to test. But I can try to come up with something.

Reply:
@TomasTomecek Test [added](https://github.com/packit/packit/pull/1680/files#diff-a28b281ee0902c932527e269c585ac109bdd98aa5ceebe05ca97a10a5a432651R558-R676), do you think it makes sense?